purescript-mode.el

```diff
@@ -11,6 +11,8 @@
 ;;          2014      Tim Dysinger <tim@dysinger.net>
 ;; Keywords: faces files PureScript
 ;; URL: https://github.com/purescript-emacs/purescript-mode
+;; Package-Version: @VERSION@
```

Comment:
Is `@VERSION@` a correct value? When package is installed from Melpa, I see this case value of `20240930.737`, which I presume automatically substituted. [The documentation](https://www.gnu.org/software/emacs/manual/html_node/elisp/Packaging-Basics.html) doesn't seem to mention such value.